src/Assetic/Extension/Twig/TwigResource.php

```diff
@@ -32,7 +32,7 @@ public function __construct(\Twig_LoaderInterface $loader, $name)
     public function getContent()
     {
         try {
-            return $this->loader->getSource($this->name);
+            return $this->loader->getSourceContext($this->name);
```

Comment:
I had no better solution - as we conflict with twig<1.27 I noticed that every implementation of `Twig_LoaderInterface` also implements `Twig_SourceContextLoaderInterface` which has `getSourceContext`. About returning type I was thinking about adding extra method in `TwigResource.php`, but this was simpler.